src/pcre2_compile.c

```diff
@@ -2835,8 +2838,8 @@ be quantified. */
 
 /* Here's the actual function. */
 
-static int parse_regex(PCRE2_SPTR ptr, uint32_t options, BOOL *has_lookbehind,
-  compile_block *cb)
+static int parse_regex(PCRE2_SPTR ptr, uint32_t options, uint32_t xoptions,
```

Comment:
We removed `(?t)` and replaced with `(*TURKISH_CASING)`, so the xoptions can't be read from the compile-context anymore (since they can now be changed with a start-of-pattern option).

Reply:
I mean we should revert the change then.